docs/elasticsearch.rst

```diff
@@ -1,6 +1,56 @@
 ElasticSearch
 =============
 
+Playing Nice with ES: Simple things to know to get reasonable performance
```

Comment:
Ha, unhelpful indeed. Maybe I'll delete that doc, and move this stuff to `Querying ElasticSearch` at the bottom of this doc, like you suggested.